coreneuron/permute/cellorder.hpp

```diff
@@ -12,8 +13,8 @@ extern void solve_interleaved(int ith);
 class InterleaveInfo {
   public:
     InterleaveInfo() = default;
-    InterleaveInfo(const InterleaveInfo&);
-    InterleaveInfo& operator=(const InterleaveInfo&);
+    InterleaveInfo(const InterleaveInfo&) = delete;
+    InterleaveInfo& operator=(const InterleaveInfo&) = delete;
     virtual ~InterleaveInfo();
```

Comment:
As far as I can tell there are no derived classes here. No need to have `virtual` here (which creates the extra `vtable`)